app/src/@anchor-protocol/app-fns/forms/borrow/borrow.ts

```diff
@@ -1,4 +1,4 @@
-import { OverseerWhitelistWithDisplay } from '@anchor-protocol/app-provider';
+import { OverseerWhitelistWithDisplay } from '@anchor-protocol/app-provider/queries/borrow/utils/tokenDisplay';
```

Comment:
export these types from the index.ts in the app-provider folder so we can avoid the extended imports.

Reply:
done